src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case '': //             does it fit to checklist p.4 ?
+        return '';
```

Comment:
what is this? maby you don't need it?

Reply:
nope, it doesn't fit to checklist p4, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#:~:text=encountered%20break.-,default,-Optional